### PR TITLE
Add news model and endpoint

### PR DESCRIPTION
**Pull request by just-paja**
* Separate rest serializers to files per endpoint.
* Add News model and admin
* Add News endpoint

### CODE REVIEW

File: api/models/news.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""Import Django models."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+
+
+class News(Base):
```
Comment (PetrDlouhy) [inline api/models/news.py]: Why not inherit from `api_textual.AbstractText` and place this to `api_textual` application?

File: api/migrations/0002_news.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```
Comment (PetrDlouhy) [inline api/migrations/0002_news.py]: Have we started to create migration history now? Just asking.

File: api/models/news.py (hunk 1 above)
Comment (just-paja) [inline api/models/news.py]: Now that I know the purpose of `api_textual`, it makes sense.

File: api/migrations/0002_news.py (hunk 2 above)
Comment (just-paja) [inline api/migrations/0002_news.py]: @PetrDlouhy, yes, we need to. It's already running on staging and live database.

Comment (PetrDlouhy) [inline api/migrations/0002_news.py]: OK, I am taking that into account.

File: api/models/news.py (hunk 1 above)
Comment (just-paja) [inline api/models/news.py]: I will change it, so please do not merge yet.

**Comment by just-paja** [comment]
Fixed pylama issues

**Comment by just-paja** [comment]
And moved model to be based on `api_textual`s AbstractText

**Comment by PetrDlouhy** [comment]
Nice. We would need to rename `api.rest.serializers` to `api.rest.years`, which was part of original PR. But that could be done in different PR.

**Pull request merged**